e2e_playwright/st_expander_test.py

```diff
@@ -94,3 +96,21 @@ def test_expander_session_state_set(app: Page):
 
     expect(text_elements.nth(0)).to_have_text("0.0", use_inner_text=True)
     expect(text_elements.nth(1)).to_have_text("0.0", use_inner_text=True)
+
+
+def test_expander_renders_icon(app: Page):
+    """Test that an expander renders a material icon and an emoji icon."""
+    main_container = app.get_by_test_id("stAppViewBlockContainer")
+    main_expanders = main_container.get_by_test_id("stExpander")
```

Comment:
nit: I recently added a better method in e2e_playwright.shared.app_utils to get an expander by label (`get_expander`). This might be a more stable way compared to getting the first expander in the app view container.